packages/core/src/components/Image/index.tsx

```diff
@@ -77,6 +81,8 @@ export type Props = {
   alignTop?: boolean;
   /** Alternate text if image cannot be displayed */
   alt: string;
+  /** Sets background color to color.core.neutral[0] */
```

Comment:
small nit: would rephrase this to not use "color.core.neutral[0]", e.g. "Sets a neutral background color"